src/request/RequestApi.js

```diff
@@ -40,9 +40,9 @@ class RequestApi {
      */
     async request(request) {
         // Deprecated, only for database migration
-        // TODO Maybe only check on iOS/Safari?
-        // Would require to load the BrowserDetection class for every request.
-        await this._checkForMigrationFlag();
+        if (BrowserDetection.isIos() || BrowserDetection.isSafari() && this._hasMigrateFlag()) {
```

Comment:
Have to put (BrowserDetection.isIos() || BrowserDetection.isSafari()) into parentheses